src/app/button-group/static/icon-buttons/icon-button-group.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
```

Comment:
This is a TODO for updating the dates in our file headers.

Reply:
Yes. But I think we need to do this in one go for all files.

Reply:
Right, thats what I meant but it didn't come accross that way. I'll file an issue for doing it as a chore.